docker-compose.yml

```diff
@@ -34,7 +33,8 @@ services:
   selenium:
     image: selenium/standalone-chrome:4.1.3-20220405
     environment:
-      - DISPLAY=:99
+      - SCREEN_WIDTH=1280
+      - SCREEN_HEIGHT=800
```

Comment:
TBD.\
Setting this explicitly should make tests more stable that test element positions and sizes.\
I also found this in other packages.

For the removed DISPLAY=:99 this is about a display port, but I see other projects that removed it, and seems not needed.